### PR TITLE
add testcase for symbolic conversion

### PR DESCRIPTION
**Pull request by cqc-melf**
# Description

Adding tests for https://github.com/CQCL/pytket-qiskit/pull/314 

# Related issues

Please mention any github issues addressed by this PR.

# Checklist

- [ ] I have performed a self-review of my code.
- [ ] I have commented hard-to-understand parts of my code.
- [ ] I have made corresponding changes to the public API documentation.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have updated the changelog with any user-facing changes.

### CODE REVIEW

File: tests/qiskit_convert_test.py [hunk 1]
```diff
@@ -1091,3 +1094,24 @@ def test_RealAmplitudes_numeric_params() -> None:
     unitary2 = tkc2.get_unitary()
     assert compare_unitaries(qiskit_unitary, unitary1)
     assert compare_unitaries(unitary1, unitary2)
+
+
+# https://github.com/CQCL/pytket-qiskit/issues/256
+def test_symbolic_param_conv() -> None:
```
Comment (CalMacCQ) [inline tests/qiskit_convert_test.py:1100]: I note that this test does not have an assert statement or anything of that kind.

Is the idea just to ensure that this code executes successfully and doesn't give the 
```
TypeError: EvalfMixin.evalf() got an unexpected keyword argument 'real'
```

As before?

**Review by CalMacCQ — COMMENTED** [review]
One clarification needed. Also can you link to the orignal PR in the desciption please?

File: tests/qiskit_convert_test.py (hunk 1 above)
Comment (cqc-melf) [reply tests/qiskit_convert_test.py:1100]: Yes that was my idea, I was thinking about checking stuff in the final circuit, but I would expect the way qiskit is setting up the circuit or the compilation works might change without generating any issues here, so I decided not to do any additional checks.

**Comment by cqc-melf** [comment]
> One clarification needed. Also can you link to the orignal PR in the desciption please?

Done

**Review by CalMacCQ — APPROVED** [review]

**Pull request merged**